### PR TITLE
ANT-2878

### PR DESCRIPTION
**Pull request by eleon-chata-ai**
change-warning-icon-size-and-color

### CODE REVIEW

File: src/components/Notifications/DataAlerts/DataAlertListItem/DataAlertListItem.scss [hunk 1]
```diff
@@ -214,9 +214,15 @@
...
+      &.data-alert-warning {
+        font-size: x-large;
+        color: var(--react-autoql-warning-color);
+        opacity: 1;
+      }
+
       &.data-alert-error {
         font-size: x-large;
-        color: #f04438;
+        color: var(--react-autoql-error-color); // #f04438;
```
Comment (Shiv-Chata) [inline src/components/Notifications/DataAlerts/DataAlertListItem/DataAlertListItem.scss:225]: Remove comment at end of line

File: src/components/Notifications/DataAlerts/DataAlertListItem/DataAlertListItem.scss [hunk 2]
```diff
@@ -242,9 +248,14 @@
...
 
+      &.data-alert-warning {
+        cursor: pointer;
+        color: var(--react-autoql-warning-color);
+      }
+
       &.data-alert-error {
         cursor: pointer;
-        color: #f04438;
+        color: var(--react-autoql-error-color); // #f04438;
```
Comment (Shiv-Chata) [inline src/components/Notifications/DataAlerts/DataAlertListItem/DataAlertListItem.scss:258]: Remove comment at end of line

**Review by Shiv-Chata — COMMENTED** [review]
Left some comments

File: src/components/Notifications/DataAlerts/DataAlertListItem/DataAlertListItem.scss (hunk 1 above)
Comment (eleon-chata-ai) [reply src/components/Notifications/DataAlerts/DataAlertListItem/DataAlertListItem.scss:225]: removed

File: src/components/Notifications/DataAlerts/DataAlertListItem/DataAlertListItem.scss (hunk 2 above)
Comment (eleon-chata-ai) [reply src/components/Notifications/DataAlerts/DataAlertListItem/DataAlertListItem.scss:258]: removed

**Review by Shiv-Chata — APPROVED** [review]

**Pull request merged**